src/App.js

```diff
@@ -1,5 +1,20 @@
-const App = () => {
-  return <div className="App"></div>;
+import Homepage from "./pages/homepage";
+import { Nav } from "./pages/navigation";
+import GlobalStyle from "./style/globalStyle";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+
+function App() {
+  return (
+    <>
+      <Router>
+        <Nav/>
+        <GlobalStyle />
+          <Routes>
+           <Route exact path='/space-tourism-website-react-NEW' element={<Homepage />}></Route>
```

Comment:
don't you think this `url` is not the best? 😅

Reply:
Definitely 🙈 corrected